projects/saas-ui/src/app/main/components/manage-plans/manage-plans.component.ts

```diff
@@ -1,15 +1,15 @@
 import {Component, OnInit} from '@angular/core';
 import {ActivatedRoute, Router} from '@angular/router';
 import {ColDef, GridApi} from 'ag-grid-community';
-import {OnBoardingService} from '../../on-boarding/on-boarding-service';
+import {OnBoardingService} from '../../../on-boarding/on-boarding-service';
```

Comment:
not pointing everywhere 
please rectify this at all places

Reply:
done the changes everywhere